charts/kcp-operator/.helmignore

```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
+# This supports shell glob matching, relative path matching, and
+# negation (prefixed with !). Only one pattern per line.
+.DS_Store
```

Comment:
This should be in your system's ignore file and not burden every other non-Mac user.